modules/GenericDataParser/src/client/GenericDataParser.coffee

```diff
@@ -8,27 +8,3 @@ class GenericDataParserController extends BasicFileValidateAndSaveController
 		@loadImagesFile = true
 		super()
 		@$('.bv_moduleTitle').html("Simple #{window.conf.experiment.label} Loader")
-
```

Comment:
Will revert before committing. I branched from an earlier point when this wasn't fixed and accidently committed here.